delivery_cttexpress/models/delivery_carrier.py

```diff
@@ -43,6 +44,13 @@ class DeliveryCarrier(models.Model):
         string="Document format",
     )
     cttexpress_document_offset = fields.Integer(string="Document Offset")
+    cttexpress_label_delay = fields.Integer(
+        string="Shipping Label Delay",
+        help="The server might not be ready to deliver the label right after the "
+        "shipping is saved. We need to introduce a little delay. Tipacally 1s will be "
+        "enough",
+        default=0,
```

Comment:
Maybe a saner default?

```suggestion
        default=1,
```